src/main/java/org/stellar/sdk/SequentialSequenceNumberStrategy.java

```diff
@@ -0,0 +1,13 @@
+package org.stellar.sdk;
+
+public class SequentialSequenceNumberStrategy implements SequenceNumberStrategy{
+    @Override
+    public long getSequenceNumber(TransactionBuilderAccount account) {
+        return account.getIncrementedSequenceNumber();
+    }
+
+    @Override
+    public void updateSourceAccount(long newSequnceNumber, TransactionBuilderAccount account) {
+        account.incrementSequenceNumber();
```

Comment:
Hmm, shouldn't this set the account sequence number to `newSequenceNumber`? Incrementing might not be the same.

Reply:
it could, unlikely that account changes state between these two callback methods, this was refactored from what Transaction.Builder.build() did, which was `account.getIncrementedSequenceNumber()` , build tx, then `account.incrementSequenceNumber()`

Reply:
followed your suggestion, it demonstrates using the callback better. [4d11a21edab2](https://github.com/stellar/java-stellar-sdk/pull/415/commits/4d11a21edab2760f31d55f45e5de114a55da728a)